### ISSUE TITLE
a11y testing with Axe

### ISSUE THREAD

**Issue by david-crespo** [oxidecomputer/console#1110]
Should be pretty easy. We can loop through a list of URLs and run the check on each.

https://github.com/dequelabs/axe-core-npm/blob/develop/packages/playwright/README.md

**Comment by david-crespo** [comment]
I don't think we need to put this in Playwright, but it is very handy to go over the site periodically with the [Chrome extension](https://chromewebstore.google.com/detail/axe-devtools-web-accessib/lhdoppojpmngadmnindnejefpokejbdd?utm_source=deque.com&utm_medium=referral&utm_campaign=axe_hero&pli=1).

**Comment by david-crespo** [comment]
https://slack.engineering/automated-accessibility-testing-at-slack/
